<?php

	function throw_exception(){
		throw new \Exception('test exception!');
	}
	
	throw_exception();